examples/simple/main.go

```diff
@@ -56,9 +72,10 @@ func main() {
 		}
 	}
 
-	// Wait for the offer to be pasted
-	offer := webrtc.SessionDescription{}
-	signal.Decode(signal.MustReadStdin(), &offer)
+	// Tweak transceiver direction to work with Firefox
+	for _, t := range peerConnection.GetTransceivers() {
+		t.Direction = webrtc.RTPTransceiverDirectionSendonly
```

Comment:
Ah, this is new to me too. I didn't know that Firefox requires the directions. Thanks